[Bug] Build fails with TS error in ZardInputGroupComponent

## 🐞 Bug Description

The application fails to build due to a TypeScript error in the `ZardInputGroupComponent`. The component uses an incorrect import path for the `ClassValue` type which prevents the project from compiling

## 🔄 Steps to Reproduce

Steps to reproduce the behavior:

1.  Clone the repository.
2.  Install deps using `npm install`
3.  Run the dev server with `npm start`

## 📷 Expected Behavior

The app should compile and start successfully without any build time errors

## 🖥️ Screenshots

N/A (This is a build-time error that occurs in the terminal)

## 🏗️ Environment

-   **Zard UI Version:** `1.0.0-beta.1`
-   **Angular Version:** `20.1.8`
-   **Browser & Version:** N/A
-   **Operating System:** Linux

## 📜 Relevant Logs & Errors

```bash
[2] ✘ [ERROR] TS2307: Cannot find module 'class-variance-authority/dist/types' or its corresponding type declarations. [plugin angular-compiler]
[2] 
[2]     libs/zard/src/lib/components/input-group/input-group.component.ts:2:27:
[2]       2 │ import { ClassValue } from 'class-variance-authority/dist/types';
```

--- a/apps/web/public/installation/manual/input-group.md
+++ b/apps/web/public/installation/manual/input-group.md
@@ -2,7 +2,7 @@
 
 ```angular-ts title="input-group.component.ts" copyButton showLineNumbers
 import { booleanAttribute, ChangeDetectionStrategy, Component, computed, input, TemplateRef, ViewEncapsulation } from '@angular/core';
-import { ClassValue } from 'class-variance-authority/dist/types';
+import type { ClassValue } from 'clsx';
 
 import { inputGroupAddonVariants, inputGroupAffixVariants, inputGroupInputVariants, inputGroupVariants, ZardInputGroupVariants } from './input-group.variants';
 import { ZardStringTemplateOutletDirective } from '../core/directives/string-template-outlet/string-template-outlet.directive';

--- a/libs/zard/src/lib/components/input-group/input-group.component.ts
+++ b/libs/zard/src/lib/components/input-group/input-group.component.ts
@@ -1,9 +1,9 @@
 import { booleanAttribute, ChangeDetectionStrategy, Component, computed, input, TemplateRef, ViewEncapsulation } from '@angular/core';
-import { ClassValue } from 'class-variance-authority/dist/types';
+import type { ClassValue } from 'clsx';
 
-import { inputGroupAddonVariants, inputGroupAffixVariants, inputGroupInputVariants, inputGroupVariants, ZardInputGroupVariants } from './input-group.variants';
-import { ZardStringTemplateOutletDirective } from '../core/directives/string-template-outlet/string-template-outlet.directive';
 import { generateId, mergeClasses } from '../../shared/utils/utils';
+import { ZardStringTemplateOutletDirective } from '../core/directives/string-template-outlet/string-template-outlet.directive';
+import { inputGroupAddonVariants, inputGroupAffixVariants, inputGroupInputVariants, inputGroupVariants, ZardInputGroupVariants } from './input-group.variants';
 
 @Component({
   selector: 'z-input-group',

